nmoscommon/interfaceController.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2017 British Broadcasting Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import netifaces
+
+from nmoscommon import nmoscommonconfig
+try:
+    from pyipputils import ipphostname
+    ippUtilsAvailable = True
+except ImportError:
+    ippUtilsAvailable = False
+    ipphostname = None  # used to make test patching work
+
+
+class InterfaceController():
+
+    def __init__(self, logger):
+        self.logger = logger
+
+    def get_default_interfaces(self):
```

Comment:
It looks like this may be able to return None/False or [], but the callers of it only seem to check for None. Could it be modified to only return one of these options to make handling it simpler?